Sources/MapboxCoreNavigation/CoreFeedbackEvent.swift

```diff
@@ -2,8 +2,8 @@ import Foundation
 import MapboxDirections
 import Polyline
 
-class CoreFeedbackEvent: Hashable {
-    var id = UUID()
+public class CoreFeedbackEvent: Hashable, Codable {
+    public var id = UUID()
```

Comment:
If we’re making this property public, then we should make sure to align it to platform conventions: `identifier`.